src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -403,7 +405,20 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
                 args2.Add(cancellationToken);
             }
 
-            var result = _info.Method.Invoke(this, args2.ToArray());
+            object result = default;
+            try
+            {
+                result = _info.Method.Invoke(this, args2.ToArray());
```

Comment:
>result [](http://example.com/codeflow?start=16&length=6)

if result is null, can we convert to Blank(type)?

Reply:
updated.